installer-clientside/build.sh

```diff
@@ -0,0 +1,27 @@
+#! /bin/bash
+
+### THIS ONLY WORKS IF THE USER HAS BEEN ADDED TO THE DOCKER GROUP
+### RUNNING THIS WITH `sudo ./build.sh` RESULTS IN DUBIOUS OWNERSHIP
```

Comment:
Maybe check for root, and error out so we don't build?

```
if [ "$EUID" -eq 0 ]; then 
  echo "Please do not run as root/with sudo. Exiting..."
  exit 1
fi
```

Reply:
Great idea! Implemented with c3ab0d7